src/modules/sensors/vehicle_angular_velocity/VehicleAngularVelocity.cpp

```diff
@@ -377,6 +377,23 @@ void VehicleAngularVelocity::ParametersUpdate(bool force)
 
 		_calibration.ParametersUpdate();
 
+		// IMU_GYRO_RATEMAX
+		if (_param_imu_gyro_ratemax.get() <= 0) {
+			const int32_t imu_gyro_ratemax = _param_imu_gyro_ratemax.get();
+			PX4_WARN("IMU_GYRO_RATEMAX invalid (%" PRId32 "), resetting to default %" PRId32 ")", imu_gyro_ratemax,
+				 _param_imu_gyro_ratemax.get());
+			_param_imu_gyro_ratemax.reset();
```

Comment:
should the reset on line 385 go before the warning message? so that the warning prints the updated (default) value